solutions/swb-reference/integration-tests/tests/multiStep/project.test.ts

```diff
@@ -56,10 +69,16 @@ describe('multiStep project tests', () => {
       .project(createdProject.id)
       .update({ name: newName, description: newDescription }, true);
     expect(updatedProject).toMatchObject({ name: newName, description: newDescription });
+    const { data: updatedProjectAsPA } = await adminSession.resources.projects
+      .project(project1Id)
+      .update({ description: newDescription }, true);
+    expect(updatedProjectAsPA).toMatchObject({ name: getProjectAsPA.name, description: newDescription });
 
     console.log('Deleting Project');
     // eslint-disable-next-line no-unused-expressions
     expect(await adminSession.resources.projects.project(createdProject.id).delete()).resolves;
+    // eslint-disable-next-line no-unused-expressions
+    expect(await pa1Session.resources.projects.project(project1Id).delete()).resolves;
```

Comment:
Could we skip the `.resolves` check? If the were to fail, it would fail the test anyways.
Better yet, capture the delete response in a variable and then [check](https://stackoverflow.com/questions/58660449/test-the-status-code-of-a-real-request-to-an-api-with-jest) the status code, since we know what code to `expect`.